apps/src/templates/progress/ProgressLevelSet.jsx

```diff
@@ -85,6 +85,26 @@ const ProgressLevelSet = React.createClass({
     levels: PropTypes.arrayOf(levelType).isRequired
   },
 
+  getIcon() {
+    const { levels } = this.props;
+    const level = levels[0];
+
+    // TODO - Once we know what locked levels and peer reviews are going to look
+    // like in the redesign, we'll need add logic for those here.
+
+    if (level.icon) {
+      // Eventually I'd like to have dashboard return an icon type. For now, I'm just
+      // going to treat the css class it sends as a type, and map it to an icon name.
```

Comment:
It seems like it should be possible to have a JSON file with constants in it that we parse in both rails and apps to create enums. Not sure exactly where that should live. Maybe in shared like `color.scss`. Or maybe it just lives in either dashboard and apps, and the other one knows where to find it.